plugin/core/src/test/groovy/com/novoda/gradle/release/ReleasePluginTest.groovy

```diff
@@ -95,7 +95,7 @@ class ReleasePluginTest {
 
     @Test
     void shouldPackageAllSources() {
-        GradleTruth.assumeThat(result).isSuccess()
+        GradleTruth.assumeThat(configuration.expectedBuildSuccess).isTrue()
 
         GradleTruth.assertThat(result.task(configuration.packageSourcesTaskName)).hasOutcome(TaskOutcome.SUCCESS)
```

Comment:
Out of scope for this PR, but where is `result` coming from? Is it a lazy property we define with the private getter `getResult()`?

Reply:
Yes, it is just a getter accessing a static cache of results, see [L55-L61](https://github.com/novoda/bintray-release/pull/243/files/fa789885cba26c0cb11cffff878fa517b796649c#diff-9e245dff7e5786027a225259dc0fff8bR55)

Reply:
It was introduced in #242